content/contribute/code.md

```diff
@@ -0,0 +1,124 @@
+---
+How to contribute to Wazo Platform code base
+---
+
+In order to contribute to Wazo Platform you need to be able to
+retrieve the source code, edit the code, try your changes and contribute
+the code to the Git repository.
+
+# Getting the code
+
+The source code for Wazo Platform is available on
+[GitHub](https://github.com/wazo-platform). Our GitHub organization contains
+over 200 repositories. Finding the one you want to contribute can be a
+daunting task.
+
+The [documentation page](/documentation) can help you find
+which repository you should be working on.
+[Asking for help](#asking-for-help) is always an option when looking at the
+less popular corners of the source code.
+
+You can then
+[clone](https://help.github.com/en/articles/cloning-a-repository) the
+desired repositories on you hard drive and start coding.
+
+# Editing the code
+
+Most of Wazo Platform is written in Python, our code follows the
+[PEP8](https://www.python.org/dev/peps/pep-0008/) conventions. You can
+use a tool such as [flake8](http://flake8.pycqa.org/en/latest/) to
+validate that you code respects the standards. Some repositories also
+include the appropriate configuration to check your code using the tox
+command `tox -e linters`. Mode details in the [Style Guide](/contribute/style_guide)
+
+Follow the [Guidelines](/contribute/guidelines) to create or improve services.
+
+Respecting coding standards is not sufficient to warrant quality code.
+Your contribution should not break any existing tests and when possible,
+it should add tests for the code you are adding. We use 3 kind of tests.
+Unittests, Integration tests and acceptance tests.
+
+## Unittests
+
+Unittests are small tests that exercise a function or method in your
+code. These tests should be fast and should not depend on other services
+running on your system, such as a database. It should also leave your
+environment in the same state, no files laying around.
+
+You can execute unittests with the following command
+
+```ShellSession
+$ tox -epy37
+```
+
+## Integration tests
+
+Integration tests exercise a service as a black box. It uses the public
+API of the service and use the API to assert that the test passes. Our
```

Comment:
`It uses the public API of the service and use the API to assert that the test passes` =>
`They use the public API of the service to assert that the tests pass` or `Each test uses the public API of the service to assert that it passes.`